BioForm/profiles/models.py

```diff
@@ -1,9 +1,85 @@
-from django.contrib.auth.models import User
+from tkinter import Widget
```

Comment:
Why do we need to import it?

Reply:
I was trying a solution for datetime that included the Widget. This got included automatically so I didn't notice it's there. I did not go with that solution so I'll remove the extra imports.